CMakeLists.txt

```diff
@@ -22,10 +22,10 @@ include_directories(
   ${catkin_INCLUDE_DIRS}
 )
 
-add_library(pointcloud_to_laserscan src/pointcloud_to_laserscan_nodelet.cpp)
+add_library(pointcloud_to_laserscan src/ipa_pointcloud_to_laserscan_nodelet.cpp)
```

Comment:
well as you added the "ipa" variant as new files, you should add new `add_library` and `add_executable`, too!
otherwise the original files are no longer built...